test/unit/specs/api/localResolvers/loan.spec.js

```diff
@@ -237,11 +237,11 @@ describe('loan.js', () => {
 				.toBe(resolvers.LoanPartner.fundraisingTimeLeftMilliseconds);
 		});
 
-		it('Returns the time remaining before expiration in words', () => {
+		it('Returns the time remaining before expiration in milliseconds', () => {
 			const now = new Date();
 			const tenMinutes = add(now, { minutes: 10 });
 			const twoHours = add(now, { hours: 2 });
-			const fiveDays = add(now, { days: 5 });
```

Comment:
This test actually fails currently because of DST (Nov 6) -- switching it to add hours instead of days seems to get around this.

Reply:
Nice, thanks for the fix, I just ran into this one as well!